ReactAndroid/src/main/java/com/facebook/react/modules/debug/SourceCodeModule.java

```diff
@@ -35,14 +35,14 @@ public SourceCodeModule(String sourceUrl, String sourceMapUrl) {
 
   @Override
   public String getName() {
-    return "RKSourceCode";
+    return "RCTSourceCode";
```

Comment:
I'm not sure about this. I searched the codebase for `RKSourceCode` and it's not used anywhere. `RCTSourceCode` is used in JavaScript and Obj-C, but it's not there in any Java code.